src/app/components/data/home/data-home.html

```diff
@@ -4,7 +4,7 @@
     </div>
     <div blBrowseLayoutButtons>
         <bl-button type="plain" icon="fa fa-plus spin-hover" color="light" title="Add a file group" [matMenuTriggerFor]="addFileGroupMenu"
-            [disabled]="!hasAutoStorage" [@.disabled]="true">
+            (click)="addFileGroup()" (do)="addFileGroup()" [disabled]="!hasAutoStorage" [@.disabled]="true">
```

Comment:
The `bl-button` doesn't play nicely with the `mat-menu` component out of the box when used a trigger as I couldn't activate the trigger using the keyboard (at least on macOS).

`mat-menu` expects an actual `button` component to be used as a trigger, but here `bl-button` is a `div` not a `button` element. I had to add `click` and `do` above in order to call `trigger.openMenu()` programmatically.